hazelcast-client/src/test/java/com/hazelcast/client/test/TestClientRegistry.java

```diff
@@ -139,7 +139,11 @@ protected ClientConnection createSocketConnection(Address address) throws IOExce
             try {
                 HazelcastInstance instance = nodeRegistry.getInstance(address);
                 if (instance == null) {
-                    throw new IOException("Can not connected to " + address + ": instance does not exist");
+                    instance =  getInstanceForLoopbackAddress(address);
```

Comment:
Why is change necessary in backport ? It was not there in master.

Reply:
that is probably not needed, i will double check and remove.